Assets/MixedRealityToolkit/_Core/Boundary/MixedRealityBoundarySystem.cs

```diff
@@ -66,33 +69,109 @@ public override void Reset()
             InitializeInternal();
         }
 
+        public bool Contains(Vector3 location)
+        {
+            return Contains(location, Boundary.Type.TrackedArea);
+        }
+
+        /// <inheritdoc/>
+        public bool Contains(Vector3 location, Boundary.Type boundaryType)
```

Comment:
The original `ContainsObject` method [transformed the provided location into the camera's frame of reference](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/june18_dev/Assets/HoloToolkit/Boundary/Scripts/BoundaryManager.cs#L175) before checking it. Do you do that anywhere?

This is important, because it accounts for the case where you've teleported and the boundary follows you. Without this change, the `Contains` method always checks as if the boundary is wherever it was at app launch.

It also [changed the rectangle params when they're requested](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/june18_dev/Assets/HoloToolkit/Boundary/Scripts/BoundaryManager.cs#L237).

Reply:
missed that I believe.... i'll have to fix that

Reply:
i thought i ported most of it, though... hopefully i'm not missing too much

Reply:
Should be handled now.